Added release notes for 1.8.1 version (#407)
okteto/docs: src/content/self-hosted/install/releases.mdx

id: releases
---

## 1.8.1
23 May, 2023

### Bugfixes

> - Fixed an issue deleting dev and preview environments when using custom okteto manifests. The destroy operation was not taken into account the manifest used during deployment and it wasn't executing the destroy commands defined in the manifest.

Reviewer: ```suggestion
- Fixed an issue affecting deletion of dev and preview environments when using custom Okteto manifests. The destroy operation did not evaluate the custom manifest used during deployment, which meant the destroy commands were not executed.
```

non-blocking suggestion
